mapper/tedge_mapper/src/collectd_mapper/monitor.rs

```diff
@@ -11,8 +11,8 @@ use mqtt_client::{QoS, Topic, TopicFilter};
 const DEFAULT_HOST: &str = "localhost";
 const DEFAULT_PORT: u16 = 1883;
 const DEFAULT_MQTT_CLIENT_ID: &str = "collectd-mapper";
-const DEFAULT_BATCHING_WINDOW: u32 = 200;
-const DEFAULT_MAXIMUM_MESSAGE_DELAY: u32 = 50;
+const DEFAULT_BATCHING_WINDOW: u32 = 500;
+const DEFAULT_MAXIMUM_MESSAGE_DELAY: u32 = 500;
```

Comment:
You really wanted a max message delay of `500` or `200`? One of your earlier comments mentioned 200. So just checking.

Reply:
Oops